_pages/en_US/installing-boot9strap-(mset9-cli).txt

```diff
@@ -55,16 +49,40 @@ In this section, you will prepare the SD card data necessary for the MSET9 explo
 1. Type the number corresponding to your console model and version, then press Enter
     + The window should change to this:
 
-    ![Image: MSET9 setup](/images/screenshots/mset9/mset9-setup.png)
+    ![Image: MSET9 setup](/images/screenshots/mset9/mset9-setup-notcreated.png)
     {: .notice--info}
 
     + Ensure that the correct console model and version is displayed
-1. Type `1`, then press Enter to ensure your console is prepared for MSET9
+1. Type `1`, then press Enter to begin the process of creating the MSET9 ID1, then type `1` again and press Enter to accept the disclaimer
     + If you get an error, check the [troubleshooting](troubleshooting#installing-boot9strap-mset9), then try again
-    + If you get "Error 10: Database(s) malformed or missing!", follow the instructions [here](/images/screenshots/database-reset.jpg), then go back to Step 7 of this section
-1. If you see the message "Everything appears to be functional!", press Enter to close the MSET9 script
+1. If you see the message "Created hacked ID1.", press Enter to close the MSET9 script
 1. Reinsert your SD card into your console
 1. Power on your console
+1. Open Mii Maker
+1. Wait for your console to reach the [Welcome to Mii Maker](/images/screenshots/mset9/mii-welcome.png) screen, then exit Mii Maker and return to the Home Screen
+    + You should see [this screen](/images/screenshots/mset9/mii-extdata.png), which indicates the necessary data has been created
+    + If you just reach the [normal](/images/screenshots/mset9/mii-existing.png) Mii Maker screen, exit Mii Maker and continue to the next step
+1. Launch System Settings and navigate to `Data Management` -> `Nintendo 3DS` -> `Software` and press Reset ([image](/images/screenshots/database-reset.jpg))
+    + This will not actually wipe your data
+1. Power off your console without leaving System Settings
+1. Insert your SD card into your computer
+1. Run the MSET9 script:
+
+    + **Windows**: Double-click `mset9.bat`
+    + **macOS**: Double-click `mset9.command` and enter your password if prompted
+    + **Linux**: open a Terminal window, `cd` to the root of your SD card, then type `python3 mset9.py` and press Enter
+
+1. Type the number corresponding to your console model and version, then press Enter
+    + The window should change to this and display `Ready`:
+
+    ![Image: MSET9 setup](/images/screenshots/mset9/mset9-ready.png)
+    {: .notice--info}
+
+    + If the window says [Not ready - check MSET9 status for more details](/images/screenshots/mset9/mset9-not-ready.png)
+        + Type `2` then press enter to check the MSET9 status and follow the directions indicated, for more information check the [troubleshooting](troubleshooting#installing-boot9strap-mset9) page
```

Comment:
Type `2`, then press Enter to check the MSET9 status and follow the directions indicated. For more information, check the [troubleshooting](#) page